core/src/utils.rs

```diff
@@ -6,7 +6,7 @@ use primitive_types::U256;
 pub enum Sign {
 	Plus,
 	Minus,
-	NoSign,
+	No,
```

Comment:
Is this also related to clippy errors?

Reply:
Probably https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names

Reply:
What @birchmd. Just something that clippy was catching.

Reply:
@joshuajbouw I know. @sorpaas asked about it above and I was taking a guess at what the specific clippy warning might have been.